erpnext/portal/product_configurator/utils.py

```diff
@@ -101,7 +101,7 @@ def get_products_html_for_website(field_filters=None, attribute_filters=None):
 	return html
 
 def set_item_group_filters(field_filters):
-	if 'item_group' in field_filters:
+	if field_filters is not None and 'item_group' in field_filters:
```

Comment:
`if field_filters and 'item_group' in field_filters:`

Reply:
This is correct but increases ambiguity hence the `None` check.